test/BinTestCases.test.js

```diff
@@ -4,46 +4,55 @@
 const should = require("should");
 const path = require("path");
 const fs = require("fs");
-const childProcess = require("child_process");
+const defaultArgs = require("./binCases/ARGUMENTS.json");
```

Comment:
What I mean was a per test arguments file. You should be able to test different combinations of arguments.